src/Pages/Platforms/observationsWind.tsx

```diff
@@ -1,15 +1,18 @@
 /**
  * Specialty Wind observation page.
  */
-import * as React from "react"
+import React from "react"
 
 import { ErddapWindObservedCondition } from "Features/ERDDAP"
 import { RenderProps } from "Features/ERDDAP/Platform/Grabber"
+import { useUnitSystem } from "Features/Units"
 
 /**
  *
  * @param platform Platform object to load wind data from
  */
-export const WindObservationsPage: React.SFC<RenderProps> = ({ platform, unit_system }) => (
-  <ErddapWindObservedCondition platform={platform} unit_system={unit_system} />
-)
+export const WindObservationsPage: React.SFC<RenderProps> = ({ platform }) => {
+  const unit_system = useUnitSystem()
```

Comment:
Codacy found an issue: [Identifier 'unit_system' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)